triciaswift/tastevine-client
# Favorite List View

## To Test
1. Login to the app
2. Click on the Favorites tab in the navbar
3. A list of the current users favorites should appear.
